src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -11,6 +12,42 @@ const Body = BaseForm.extend({
     return loc('oie.next.button', 'login');
   },
 
+  initialize () {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    const policy = this.getPasswordPolicy();
+    this.displayPasswordPolicy(policy);
+  },
+
+  displayPasswordPolicy (policy) {
+    if (policy) {
+      const rulesList = getPasswordComplexityDescriptionForHtmlList( policy );
+      this.add(
+        View.extend({
+          tagName: 'section',
+          className: 'password-authenticator--rules',
+          template: 
+            `<div class="password-authenticator--heading">
+              {{i18n code="password.complexity.requirements.header" bundle="login"}}
+            </div>
+            <ul class="password-authenticator--list">
+              {{#each rulesList}}<li>{{this}}</li>{{/each}}
+            </ul>`,
+          getTemplateData: () => ({ rulesList }),
+        }),
+        {
+          prepend: true,
+          selector: '.o-form-fieldset-container',
+        }
+      );
+    }
+  },
+
+  getPasswordPolicy () {
+    // This will be overridden by password expired and password will expire soon
+    // scenarios since the policies could be different for those.
+    return null; 
```

Comment:
Instead of this, you can throw an error.

https://stackoverflow.com/questions/29480569/does-ecmascript-6-have-a-convention-for-abstract-classes

```
throw new TypeError("Must override method");
```

Reply:
This will be changed when I work on password requirements for enrollment, I'll address it then

Reply:
it is not strictly an abstract function which doesn't have default implementation.
here `getPasswordPolicy` has default implementation (return null) because EnrollAuthenticatorPassword doesn't support password requirement yet.

if we throw error, we have to `try/catch` where this function got called. feels complicated use case.